### DISCUSSION TITLE
property variant does not exist on type of select

### DISCUSSION THREAD

**Discussion by aditya-shrivastavv** [csandman/chakra-react-select#266] [Q&A]
```jsx
<Select
        isMulti
        variant={'filled'}
        name={name}
        ref={ref}
        onChange={onChange as (...event: any[]) => void}
        onBlur={onBlur}
        value={value}
        {...props}
      />
```

---

### Error

Type '{ shouldUnregister?: boolean | undefined; defaultValue?: string | boolean | ClassesGroup | ClassesGroup[] | undefined; colorScheme?: string | undefined; ... 85 more ...; ref: RefCallBack; }' is not assignable to type 'IntrinsicAttributes & Omit<PublicBaseSelectProps<unknown, boolean, GroupBase<unknown>>, StateManagedPropKeys> & Partial<...> & StateManagerAdditionalProps<...> & RefAttributes<...>'.

  **Property 'variant' does not exist on type** 'IntrinsicAttributes & Omit<PublicBaseSelectProps<unknown, boolean, GroupBase<unknown>>, StateManagedPropKeys> & Partial<...> & StateManagerAdditionalProps<...> & RefAttributes<...>'.ts(2322)

**Comment by csandman** [comment]
What version are you on?
